Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/components/speedInput/index.tsx

```diff
@@ -1,9 +1,10 @@
-const SpeedInput = () => {
+const SpeedInput = (props) => {
   return (
     <>
       <div className="flex w-full flex-col items-center gap-2">
         <label>Speed (km/h):</label>
         <input
```

Comment:
In App.tsx, you give the battery an intial value of null, but the input field is of type "number". This can cause a type error in the future.